Mathlib/Topology/Compactness/Compact.lean

```diff
@@ -26,49 +26,51 @@ We define the following properties for sets in a topological space:
 -/
 open Set Filter Topology TopologicalSpace Classical
 
+set_option autoImplicit true
```

Comment:
```suggestion
set_option autoImplicit true
```
Please don't do that. It just makes things more unpredictable and harder to read.